dev/com.ibm.ws.microprofile.config.1.1_fat/test-applications/classLoaderCache.war/src/com/ibm/ws/microprofile/appConfig/classLoaderCache/test/ClassLoaderCacheTestServlet.java

```diff
@@ -68,4 +70,24 @@ private static ClassLoader getRootClassLoader() {
         System.out.println("Root ClassLoader: " + rootCL);
         return rootCL;
     }
+
+    private int getConfigCacheSize(ConfigProviderResolver resolver) throws IllegalAccessException, IllegalArgumentException, InvocationTargetException, NoSuchFieldException, SecurityException {
+        int size = -1;
+        try {
+            Method getConfigCacheSizeMethod = resolver.getClass().getMethod("getConfigCacheSize");
+            size = (int) getConfigCacheSizeMethod.invoke(resolver);
+        } catch (NoSuchMethodException e) {
+            Field configsForClassLoaderField = resolver.getClass().getSuperclass().getDeclaredField("configsForClassLoader");
+            configsForClassLoaderField.setAccessible(true);
+            @SuppressWarnings("unchecked")
+            Map<ClassLoader, Config> configsForClassLoader = (Map<ClassLoader, Config>) configsForClassLoaderField.get(resolver);
+            size = configsForClassLoader.size();
```

Comment:
Can we have comments so that we know which implementation classes this reflection is designed to act upon?